nuget.config

```diff
@@ -10,5 +10,10 @@
    </config>
    <packageSources>
         <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+        <add key="Fast-Blazor-Feed" value="https://pkgs.dev.azure.com/microsoftdesign/FAST/_packaging/Fast-Blazor-Feed/nuget/v3/index.json" />
    </packageSources>
+   <packageSources>
+        <clear />
+        <add key="github" value="https://nuget.pkg.github.com/OWNER/index.json" />
```

Comment:
Replace "OWNER" with Microsoft.  But, you also can simply add "github" to the `dotnet nuget push --source github` as shown in the documentation.